src/bubble/login.clj

```diff
@@ -24,6 +24,21 @@
       (handler (assoc req :current-user user))
       (login-error-redirect "You must be logged in to see that"))))
 
+(defn join-bubble-form [req]
+    (let [uri (get-in req [:uri])]
+    (println (str "HEEEEEERRRRR: " uri)))
+    (let [error (get-in req [:params :error])]
+    (views/base-view [[:h1 "Login to join bubble"]
+                      (when error [:p (str "Error: " error)])
+                      [:form {:action "/login" :method "post"}
+                       [:input {:name "phone" :placeholder "Phone #"}]
+                       [:input {:name "short-code" :type "checkbox"}]
+                       [:label {:for "short-code"} "Send a code instead"]
+                       [:button {:name "submit"} "Send me a link"]]])))
```

Comment:
If you agree with the flow I propose in this comment: https://github.com/jackrr/bubble/pull/3#issuecomment-846421253, you might want to change this page to be more of a "splash page" for the bubble that includes a button/link to join (also show bubble name, some "you have been invited blah blah blah" text). Clicking the button/link will navigate to `POST /bubbles/:id/join` -- if you're already logged in, you get added. If not, you get redirected to the existing login flow.